docs/showcase/pong/README.md

```diff
@@ -2,13 +2,13 @@
 
 ![](./pong.png)
 
-Practically the "Hello World!" of games. Pong has been remade thousands of times. I know Pong. You know Pong. We all know Pong. That being said, this time I wanted to put a little more effort than most people do. This showcase has a basic menu system, sounds, and different game states.
+Practically the "Hello World!" of games. Pong has been remade thousands of times. I know Pong. You know Pong. We all know Pong. That being said, this time I wanted to put in a little more effort than most people do. This showcase has a basic menu system, sounds, and different game states.
 
 The architecture is not the best as I prescribed to the "get things done" mentality. If I were to redo this project, I'd change a lot of things. Regardless, let's get into the postmortem.
 
 ## The Architecture
 
-I was messing around with separating state from the render code. It ended up similar to an entity component system.
+I was messing around with separating state from the render code. It ended up similar to an entity-component system.
```

Comment:
This is referring to an ECS. Use of that term is all over the map, but I think `Entity Component System model` would be the best way to refer to it here. "System" is a first-class element of the pattern here.